pyproject.toml

```diff
@@ -236,6 +227,63 @@ filterwarnings = [
 # need to ensure build directories aren't excluded from recursion
 norecursedirs = []
 
+[tool.ruff.lint]
+# E, W   pycodestyle
+# F      Pyflakes
+# UP     pyupgrade
+# B      flake8-bugbear
+# SIM    flake8-simplify
+# YTT    flake8-2020
+# ASYNC  flake8-async
+# C4     flake8-comprehensions
+# I      isort
+extend-select = ["E" ,"W", "F", "UP", "B", "SIM", "YTT", "ASYNC", "C4", "I"]
```

Comment:
Can these comments be inlined against the definition? e.g.,
```
extend-select = [
    "E",  # pycodestyle
    "W",  # pycodestyle
    "F",  # pyflakes
...
```
It will take up the same amount of space, but there's immediate correspondence between definition and explanation. Same for the exception cases below.

Reply:
Definitely!